File: 389-find-the-difference/389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int charCodeS = 0;
        int charCodeT = 0;
        
        for(int i =0; i< s.length(); i++){
            charCodeS+= (int)s.charAt(i);
        }
        for(int j = 0; j<t.length(); j++) {
            charCodeT+=(int)t.charAt(j);
        }
        
        return (char)(charCodeT - charCodeS);
    }
}

File: 389-find-the-difference/389-find-the-difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        charCodeS=0
        charCodeT=0
        
        for i in range(len(s)):
            charCodeS+=ord(s[i])
            
        for i in range(len(t)):
            charCodeT+=ord(t[i])
            
        return chr(charCodeT-charCodeS)
        